### ISSUE TITLE
CatalogUpdaterLambda loads only first 25 usage plans

### ISSUE THREAD

**Issue by paulmakh** [awslabs/aws-api-gateway-developer-portal#257]
Lambda catalog-updater uses exports.gateway.getUsagePlans({}) for load usage plans. 
getUsagePlans() function has input parameter:
```
var params = {
  keyId: 'STRING_VALUE',
  limit: 'NUMBER_VALUE',
  position: 'STRING_VALUE'
};
```
By default limit number is 25. (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/APIGateway.html#getUsagePlans-property)

So currently the lambda loads only first 25 usage plans.

**Comment by paulmakh** [comment]
The same issue is in DevPortalLambda when it loads APSs and Usage Plans

**Comment by Trial-In-Error** [comment]
Ah! Thanks for the bug report. We'll take a look at implementing pagination on it. Do you currently have more than 25 usage plans, and are they all in use?

**Comment by paulmakh** [comment]
Yes, currently we have around 50 usage plans. Would you like to say that is unusual?

**Comment by Trial-In-Error** [comment]
@paulmakh, It is a little unusual! Typically, a usage plan represents either a tier of usage ("free", "paid", "premium", "enterprise") or a bundle of products ("housing APIs", "restaurant APIs", "review APIs"). Could you explain your use case a bit more to help me understand it?

**Comment by paulmakh** [comment]
@Trial-In-Error - We have few services and few consumers and would like to manage how to each consumer use each service separately.

**Comment by alex-chew** [comment]
This issue is fixed as of v3.0.2.
